backend/build.gradle

```diff
@@ -0,0 +1,82 @@
+
+plugins {
+    id 'org.springframework.boot' version '2.5.5'
+    id 'io.spring.dependency-management' version '1.0.14.RELEASE'
+    id 'java'
+    id 'jacoco'
+    id "io.freefair.lombok" version "6.5.1"
+}
+
+configurations {
+    developmentOnly
+    compileOnly {
+        extendsFrom annotationProcessor
+    }
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation 'org.springframework.boot:spring-boot-starter'
+    implementation 'org.springframework.boot:spring-boot-starter-web'
+    implementation 'org.springframework.boot:spring-boot-starter-data-mongodb'
+    implementation 'org.springframework.boot:spring-boot-starter-websocket'
+    implementation 'com.google.oauth-client:google-oauth-client:1.33.3'
+    implementation 'com.google.api-client:google-api-client-gson:1.33.0'
+    implementation 'com.google.code.gson:gson:2.8.9'
+    implementation 'io.springfox:springfox-boot-starter:3.0.0'
+    implementation 'io.springfox:springfox-swagger-ui:3.0.0'
+
+    testImplementation 'org.springframework.boot:spring-boot-starter-test'
+    testImplementation 'de.flapdoodle.embed:de.flapdoodle.embed.mongo'
+}
+
+group = 'io.diveni'
+version = '1.0.0-SNAPSHOT'
+description = 'Diveni-backend'
+java.sourceCompatibility = JavaVersion.VERSION_11
+
+tasks.withType(JavaCompile) {
+    options.encoding = 'UTF-8'
+}
+
+jar {
+    manifest {
+        attributes 'Main-Class': 'src.main.java.io.diveni.backend.DiveniApplication'
```

Comment:
should this not be without ‘src.main.java‘?

Reply:
Yes, I think you are right. In fact you do not even need this task. If you want to compile your project you call `gradle bootJar`.

That is why I removed it.